### PR TITLE
Feature/boas 153 applicant website details page

### PR DESCRIPTION
**Pull request by DavidJWhiteDev**
## Describe your changes

page for input of applicant website.
Validation if a value is entered, to ensure it is a valid URL - uses std express validator isUrl function, with an additional check for '@' sign, which inexplicably is allowed in the isUrl check.
GET and POST tests added.

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/create/applicant/_website.njk [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 {% block stepForm %}
-	work in progress
+	{{ govukInput({
+		id: "applicant.website",
+		name: "applicant.website",
+		classes: "govuk-!-width-one-third",
+		value: values['applicant.website'],
+		errorMessage: errors['applicant.website'] | errorMessage,
+		hint: {
+			text: "for example, https://www.heathrow.com"
```
Comment (francescobrenci-AR) [inline apps/web/src/server/views/applications/create/applicant/_website.njk:17]: can you make a note that the correct text is this and the design needs updating please

Comment (DavidJWhiteDev) [reply apps/web/src/server/views/applications/create/applicant/_website.njk:17]: yep, note to design was already added to ticket.

File: apps/web/src/server/applications/create/applicant/applications-create-applicant.validators.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { body } from 'express-validator';
+
+export const validateApplicationsCreateApplicantWebsite = createValidator(
+	// regex check added to exclude @ signs, which for some reason are valid in isUrl
+	body('applicant.website')
+		.optional({ checkFalsy: true })
+		.trim()
+		.matches(/^[^@]*$/)
+		.withMessage('Enter a valid website')
+		.isURL({
```
[the remark this answers is not in the document]
Comment (DavidJWhiteDev) [reply apps/web/src/server/applications/create/applicant/applications-create-applicant.validators.js:11]: yes, subdomains are valid and work correctly.

**Pull request merged**